homework/week-2/jorgereina1986/Horoscope/app/src/main/java/jorgereina1986/c4q/nyc/horoscope/GameActivity.java

```diff
@@ -0,0 +1,63 @@
+package jorgereina1986.c4q.nyc.horoscope;
+
+import android.support.v7.app.ActionBarActivity;
+import android.os.Bundle;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.widget.TextView;
+
+import java.util.Calendar;
+import java.util.Random;
+
+
+public class GameActivity extends ActionBarActivity {
+
+    TextView randomDate;
+    int randomMonth;
+    int randomDay;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_game);
+    }
+
+    public static String Question() {
```

Comment:
I think this method could have a more appropriate name. You are returning a date, maybe something like getDate.